corporate/lib/stripe.py

```diff
@@ -22,24 +23,37 @@
 from django.utils.translation import override as override_language
 from typing_extensions import ParamSpec, override
 
+from corporate.lib.support import get_support_url
```

Comment:
This is an import cycle. I think the best plan might be to just import this locally in the function using it with a TODO comment and refactor later.

Reply:
Thanks! Done.